src/EnergyPlus/MixedAir.cc

```diff
@@ -2364,6 +2406,17 @@ void ProcessOAControllerInputs(EnergyPlusData &state,
         }
     }
 
+    if (NumAlphas > 19) {
+        if (!lAlphaBlanks(20)) {
+            if (UtilityRoutines::SameString(AlphArray(20), "EconomizerFirst")) {
+                state.dataMixedAir->OAController(OutAirNum).EconomizerStagingType = DataHVACGlobals::EconomizerStagingType::EconomizerFirst;
+            } else {
+                state.dataMixedAir->OAController(OutAirNum).EconomizerStagingType =
+                    DataHVACGlobals::EconomizerStagingType::InterlockedWithMechanicalCooling;
+            }
+        }
+    }
+
```

Comment:
AlphArray(20) could be blank since it's not required. But if (NumAlphas > 19) then that field is present and the default is InterlockedWithMechanicalCooling

    A20; \field Economizer Operation Staging
       \type choice
       \key EconomizerFirst
       \key InterlockedWithMechanicalCooling
       \default InterlockedWithMechanicalCooling        

And this variable is initialized to interlock so this looks OK (I thought this was wrong until I looked at this).

    struct OAControllerProps

    DataHVACGlobals::EconomizerStagingType EconomizerStagingType =
            DataHVACGlobals::EconomizerStagingType::InterlockedWithMechanicalCooling; // User input select